packages/cadl-ranch-specs/http/azure/core/lro/rpc/main.tsp

```diff
@@ -95,4 +95,15 @@ Expected response body:
 ```
 """)
 @route("/generations:submit")
-op longRunningRpc is global.Azure.Core.LongRunningRpcOperation<GenerationOptions, GenerationResponse, GenerationResult>;
+op longRunningRpc is global.Azure.Core.LongRunningRpcOperation<
+  BodyParameter<GenerationOptions>,
+  GenerationResponse,
+  GenerationResult
+>;
+
+alias BodyParameter<T, TName extends valueof string = "body", TDoc extends valueof string = ""> = {
+  @doc(TDoc)
+  @friendlyName(TName)
+  @bodyRoot
+  body: T;
+};
```

Comment:
switch to use local `BodyParameter` in azure.

will switch to `Azure.Core.BodyParameter` in future.